react/components/results.js

```diff
@@ -21,15 +21,29 @@ const ResultsComponent = React.createClass({
       console.log('currentBlockId', currentBlockId);
       // point to blockbuilder graph search prototype
       // running on digital ocean server
-      window.open(`http://138.197.194.92:8080/?gist_id=${currentBlockId}`, '_blank');
+      window.open(
+        `http://138.197.194.92:8080/?gist_id=${currentBlockId}`,
+        '_blank'
+      );
     });
   },
   onMouseOut() {
     Mousetrap.unbind('shift');
   },
   render() {
     var query = this.props.query;
-    var results = this.props.results;
+    console.log('query', query);
```

Comment:
And Again?

Reply:
oh, this isn’t, intentional.  good find, will fix!

Reply:
ok, it looks like this is also a prettier formatting change